languages/es.json

```diff
@@ -0,0 +1,208 @@
+    "Unknown Error trying to open channel!": "Error desconocido intentando abrir el canal",
+    "Open Channel": "Abrir canal",
+    "Are you sure you want to open channel with partner {{destination}} using token {{tokenName}}?": "Est'a seguro que desea abrir un canal con el partner {{destination}} usando el token {{tokenName}}",
+    "Opening channel\nPlease wait, this operation could take around 4 minutes": "Abriendo canal\nPor favor, espere,\nesta operación puede tomar alrededor de 4 minutos",
+    "You need to select a token and put the partner and amount first.": "Primero debes seleccionar token, partner y monto",
+    "Lumino Channels": "Canales de Lumino",
+    "owned channels": "canales propios",
+    "nodes": "nodos",
+    "channels": "canales",
+    "Subdomains": "Subdominios",
+    "Renew Domain": "Renovar Dominio",
+    "Pay": "pAGAR",
+    "Sell it on MKP": "Venderlo en mkp",
+    "Sell it on Marketplace": "Venderlo en el Marketplace",
+    "Exchange Domain": "Intercambiar dominio",
+    "Transfer": "Transferir",
+    "Transfer Domain": "Transferir Dominio",
+    "No available options": "No hay opciones disponibles",
+    "Domains with less than 5 characters are blocked.": "No se permiten dominios con menos de 5 caracteres",
+    "Details retrieved": "Detalles obtenidos",
+    "An error happend trying to get details from domain, please try again later.": "Un error ocurrió intenando obtener los detalles del dominio. Por favor, intente más tarde",
+    "Search for domains": "Buscar dominios",
+    "Search for network": "Buscar redes",
+    "No subdomains found": "No se encontraron subdominios",
+    "Add Subdomain": "Agregar Subdominios",
+    "Your balance:": "Tu balance:",
+    "Wait please...": "Por favor espere...",
+    "Renew now!": "Renovar ahoras",
+    "Error trying to update configuration: {{errorMessage}}": "Error intentando actualizar la configuracción: {{errorMessage}}",
+    "Lumino networks directory": "Directorio de redes de Lumino",
+    "Network Results": "Resultados de la Red",
+    "My Lumino Networks": "Mis redes de Lumino",
+    "Lumino networks available": "Redes de Luminos disponibles",
+    "Exchange": "Intercambiar",
+    "Add funds": "Agregar fondos",
+    "Payment Sent": "Pago Envíado",
+    "Payment Delivered": "Pago Entregado",
+    "Unknown Error trying to pay!": "Error desconocido intentando pagar",
+    "¿Are you sure you want to pay {{paymentInput}} tokens to partner {{partnerAddress}}?": "¿Está seguro que desea pagarle {{paymentInput}} tokens al partner {{partnerAddress}}?",
+    "Paying...": "Pagando...",
+    "Closing channel\nPlease wait, this operation could take around 4 minutes": "Cerrando canal\nPor favor, espere,\n Esta  oepración puede tardar alrededor de 4 minutos.",
+    "Channel has been closed\nThe settlement will take around 2 hours": "El canal ha sido cerrado\nLa liquidación tomará alrededor de 2 horas",
```

Comment:
Let's say settlement instead of "Liquidacion", something like "El canal ha sido cerrado\El settlement tomará alrededor de 2 horas"